src/mgmt/srv_drpc.c

```diff
@@ -692,7 +704,7 @@ ds_mgmt_drpc_pool_extend(Drpc__Call *drpc_req, Drpc__Response *drpc_resp)
 		D_GOTO(out_list, rc = -DER_NOMEM);
 
 	rc = ds_mgmt_pool_extend(uuid, svc_ranks, rank_list, "pmem",
-				 req->scmbytes, req->nvmebytes,
+				 scm_bytes, nvme_bytes,
 				 req->n_faultdomains, req->faultdomains);
```

Comment:
So is the long-term plan to modify these APIs to accept a uint64_t array?

Reply:
I think this is planned for later.
@NiuYawei : is that right? I.e. is this /cart/ in scope of 2.0? My understanding was we were only talking about management calls and all the collective calls will be updated later.

Reply:
Yes, I think that can be done later.